Document/0x04a-Mobile-App-Taxonomy.md

```diff
@@ -1,32 +1,40 @@
+
+Native apps inherently have the capability to provide the fastest performance with the highest degree of reliability. They usually adhere to platform-specific design principles (e.g. the [Android Design Principles](https://developer.android.com/design/get-started/principles.html "Android Design Principles")), which tends to provide a more consistent user interface (UI), compared to *hybrid* or *web* apps. Due to their close integration with the operating system, native apps can directly access almost every component of the device (camera, sensors, hardware-backed key stores, etc.)
 
-Native mobile apps can provide fast performance and a high degree of reliability. They usually adhere to platform-specific design principles (e.g. the [Android Design Principles](https://developer.android.com/design/get-started/principles.html "Android Design Principles")), and provide a more consistent UI than `hybrid` and `web` apps. Due to their close integration with the operating system, native apps have access to almost every component of the device (camera, sensors, hardware backed key stores, etc.)
+With respect to the previous statement, some ambiguity exists when discussing *native apps* for Android as the platform provides two sets of APIs for development - the Android SDK and the Android NDK. The SDK (or Software Development Kit) is a Java API and the current default for developing apps. The NDK (or Native Development Kit) is a C/C++ API used for developing application components requiring specific optimization, benefiting from access to lower level APIs (such as OpenGL).
 
-There can be some ambiguity when discussing `native` apps for Android. Android provides two sets of APIs to develop against - the Android SDK and the Android NDK. The SDK (or Software Development Kit) is a Java API and is the default API against which applications are built. The NDK (or Native Development Kit) is a C/C++ based API used for developing application components that require specific optimization, or which can otherwise benefit from access to lower level APIs (such as OpenGL). Normally, you can only distribute apps built with the SDK, which potentially can also consume NDK APIs. Therefore we say that Android `native **apps**` (built with the SDK) can have `native **code**` (built with the NDK).
+Normally with Android, you can only distribute apps built with the SDK which can incorporate some developmental features from NDK APIs. Therefore, we say that Android *native apps* (i.e. built with the SDK) may have *native* code also built with the NDK. However, it should be noted that apps developed with the NDK API cannot incorporate SDK elements.
```

Comment:
I'd remove this sentence.

"However, it should be noted that apps developed with the NDK API cannot incorporate SDK elements."